### DISCUSSION TITLE
Transmitting data via WebRTC to web page

### DISCUSSION THREAD

**Discussion by DiKey98** [bluenviron/mediamtx#3052] [Questions] [answered]
### Question

There are 3 types of streams in the RTSP stream: video, audio and application (with type Generic). Can mediamtx transmitt application (Generic) data to web page via WebRTC? For example, through data channel?

**Comment by aler9** [comment, marked as answer]
Hello, no it can't but it might be a good future enhancement.

**Reply by DiKey98** [reply]
Thank you!

[a comment by an automated account is not shown]
